alchemist/.pmd

```
@@ -722,30 +658,10 @@
             <ruleset>Empty Code</ruleset>
         </rule>
         <rule>
-            <name>EmptyCatchBlock</name>
-            <ruleset>Empty Code</ruleset>
-        </rule>
```

Empty catches are TOTALLY bad.